src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheck.java

```diff
@@ -113,156 +117,98 @@
     public static final String MSG_KEY_JAVADOC_DUPLICATE_TAG =
             "javadoc.duplicateTag";
 
-    /**
-     * A key is pointing to the warning message text in "messages.properties"
-     * file.
-     */
-    public static final String MSG_KEY_JAVADOC_MISSING = "javadoc.missing";
-
     /** {@link Deprecated Deprecated} annotation name. */
     private static final String DEPRECATED = "Deprecated";
 
     /** Fully-qualified {@link Deprecated Deprecated} annotation name. */
     private static final String FQ_DEPRECATED = "java.lang." + DEPRECATED;
 
-    /** Compiled regexp to match Javadoc tag with no argument. */
-    private static final Pattern MATCH_DEPRECATED =
-            CommonUtil.createPattern("@(deprecated)\\s+\\S");
-
-    /** Compiled regexp to match first part of multilineJavadoc tags. */
-    private static final Pattern MATCH_DEPRECATED_MULTILINE_START =
-            CommonUtil.createPattern("@(deprecated)\\s*$");
-
-    /** Compiled regexp to look for a continuation of the comment. */
-    private static final Pattern MATCH_DEPRECATED_MULTILINE_CONT =
-            CommonUtil.createPattern("(\\*/|@|[^\\s\\*])");
-
-    /** Multiline finished at end of comment. */
-    private static final String END_JAVADOC = "*/";
-    /** Multiline finished at next Javadoc. */
-    private static final String NEXT_TAG = "@";
-
-    /**
-     * Ignore cases when JavaDoc is missing, but still warns when JavaDoc is present but either
-     * &#64;deprecated is missing from JavaDoc or &#64;Deprecated is missing from the element.
-     */
-    private boolean skipNoJavadoc;
-
     /**
      * Setter to ignore cases when JavaDoc is missing, but still warns when JavaDoc is present
      * but either &#64;deprecated is missing from JavaDoc or &#64;Deprecated is missing
      * from the element.
      * @param skipNoJavadoc user's value of skipJavadoc
+     * @deprecated No longer valid from switch to javadoc parser.
      */
+    @Deprecated
```

Comment:
(not related to this issue)
Should `AutomaticBean` issue a warning when a deprecated property is specified in the configuration?